docs/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: "vkcs"
+page_title: "Provider: VKCS"
+description: |-
+  The VKCS provider is used to interact with VKCS services.
+  The provider needs to be configured with the proper credentials before it can be used.
+---
+
+# VKCS Provider
+
+The VKCS provider is used to interact with
+[VKCS services](https://mcs.mail.ru/). The provider needs
+to be configured with the proper credentials before it can be used.
+
+Use the navigation to the left to read about the available resources.
+
+## Example Usage
+
+Terraform 1.0 and later:
+
+```terraform
+# Configure the vkcs provider
+
+terraform {
+  required_providers {
+    vkcs = {
+      source = "MailRuCloudSolutions/vkcs"
+      version = "~> 0.5.8"
```

Comment:
Should we start with fresh version such as 0.1.0?

Reply:
Updated
Some old examples in docs/examples may use old version, they will be updated with old provider resources pull request